org.civicrm.reqangular/src/common/modules/uib-tabset.js

```diff
@@ -0,0 +1,18 @@
+define([
+  'common/angular',
+  'common/angularBootstrap',
+  'common/modules/templates'
+], function (angular) {
+  'use strict';
+
+  return angular.module("common.uibTabset", [])
```

Comment:
We should not add this to the `common.directives` because at this moment we need to override a single directive so it looks fine, but when there will be multiple overrides the `common.directives` would get polluted easily.